recipes/windows-terminal/wt.cmd

```diff
@@ -0,0 +1 @@
+@"%~dp0..\windows-terminal\wt.exe" %*
```

Comment:
I view `.bat` and `.cmd` as pretty interchangeable. I've never encountered a situation where it mattered though the SO you linked to is pretty informative! It does seem that `.cmd` is a bit stricter and generally preferred.

My general adhoc convention is to call scripts `.bat` and reserve `.cmd` for files which just set up an environment before calling an external binary.

Since the script doesn't use any of the mentioned commands where there is a difference in errorlevel handling I think it's a moot point. If you'd prefer it be a `.bat` file I can make that change.

Reply:
> If we move everything under `Scripts` we might not need this.

If the application was under `scripts/windows-terminal` you'd still need a driver script in `scripts/` to call it without having to resort to the full pathname of the executable.